flink-runtime-web/web-dashboard/src/app/pages/job/checkpoints/job-checkpoints.component.html

```diff
@@ -550,6 +550,19 @@
               {{ checkPointConfig['checkpoints_after_tasks_finish'] ? 'Enabled' : 'Disabled' }}
             </td>
           </tr>
+          <tr>
+            <td>Changelog</td>
+            <td *ngIf="checkPointConfig['state_backend'] === 'ChangelogStateBackend'">Enabled</td>
+            <td *ngIf="checkPointConfig['state_backend'] !== 'ChangelogStateBackend'">Disabled</td>
+          </tr>
+          <tr *ngIf="checkPointConfig['state_backend'] === 'ChangelogStateBackend'">
+            <td>Changelog Storage</td>
+            <td>{{ checkPointConfig['changelog_storage'] }}</td>
+          </tr>
+          <tr *ngIf="checkPointConfig['state_backend'] === 'ChangelogStateBackend'">
+            <td>Changelog Periodic Materialization Interval</td>
+            <td>{{ checkPointConfig['changelog_periodic_materialization_interval'] }}</td>
```

Comment:
I noticed that the curent UI does not show the time unit of interval.
I think you can change this to `{{ checkPointConfig['changelog_periodic_materialization_interval'] | humanizeDuration}}`